src/main/java/com/dabsquared/gitlabjenkins/webhook/build/SCMSourceOwnerNotifier.java

```diff
@@ -0,0 +1,46 @@
+package com.dabsquared.gitlabjenkins.webhook.build;
+
+import java.net.URISyntaxException;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import org.eclipse.jgit.transport.URIish;
+
+import jenkins.plugins.git.GitSCMSource;
+import jenkins.scm.api.SCMSource;
+import jenkins.scm.api.SCMSourceOwner;
+import hudson.model.Item;
+
+import static com.dabsquared.gitlabjenkins.util.LoggerUtil.toArray;
+
+public class SCMSourceOwnerNotifier implements Runnable{
+
+    private final static Logger LOGGER = Logger.getLogger(PushBuildAction.class.getName());
+    private final Item project;
+
+    public SCMSourceOwnerNotifier(Item project) {
```

Comment:
The class only makes sense when given an SCMSourceOwner. At the call site this is already verified, so it's nicer to just allow these as projects

```suggestion
    public SCMSourceOwnerNotifier(SCMSourceOwner project) {
```